.github/workflows/run-tests.yaml

```diff
@@ -3,6 +3,12 @@ run-name: run unit tests
 on:
   pull_request:
     branches-ignore: ["release-please*"]
+    paths:
+    - "genome_kit/**"
+    - "src/**"
+    - "tests/**"
+    - "genomekit_dev.yml"
+    - "CMakelists.txt"
```

Comment:
we don't use this to build, not sure if it tests anything

Reply:
Not yet, but I think I'll start running the c++ unit tests soon.

Reply:
I'll remove for now.

Reply:
what about the dev file?

Reply:
I put it there as a proxy for conda-forge/genomekit-feedstock meta.yaml which isn't in this repo. The intent is to rerun the tests if we upgrade dependencies.
It assumes that if we update dependencies we'll update them both here and in meta.yaml.

Reply:
if someone updates the dependencies here in a feature branch, they won't be updating the meta.yaml on the conda-forge release though (since there is no release)

Reply:
True, I'll remove. Need to think how I want to address running tests on dependency upgrades *before* they hit the feedstock repo.
Or even just dependency upgrades in general.